styles/documentation.css

```diff
@@ -48,8 +57,37 @@ a{
 .listSubTopic{
     /* margin-left: 4.3rem;  */
     text-align: center;
+} 
+
+.documentContentHeader{
+    margin-bottom: 15px;
+    color: var(--primary-color);
 }
 
+.documentContentPara{
+    width: 650px; 
+    justify-content: center;
+    align-items: center;
+}
+
+.buttonVariantTitle{
+    margin-top: 5rem; 
+    color: var(--primary-color);
+}
+
+.btnVariantContainer{
+    height: 100px;
+    /* border: 2px solid black;  */
+    width: 850px;
```

Comment:
if you use relative units them is easy for responsive

Reply:
Good point, noted.